mmfunctions

```diff
@@ -148,6 +155,10 @@ _premis_event_list(){
     if [ "$eventDetail" = "makewaveform" ] ; then
         eventType="creation"
     fi
+    
+    if [ "$eventDetail" = "makefingerprint" ] ; then
+        eventType="creation"
```

Comment:
I am a feeling a little angst about this in terms of an event type, but I feel it is closer to 'creation' than 'fixity'.  Any thoughts about more suitable PREMIS glossary terms?  Or should we make up one of our own?